physionet-django/events/templates/events/email/event_cohost_cohost_status_change.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}{% autoescape off %}{% filter wordwrap:70 %}
+Dear {{ name }},
+{% if status == 'Make cohost' %}
+You have been added as Cohost to the Event : {{ event_title }}.
+
+You can now manage the event participants from your events dashboard.
+{% elif status == 'Remove cohost' %}
+Your Cohost access has been removed from the Event : {{ event_title }}.
+{% endif %}
+You can view further information about the event using the following link:
+{{ url_prefix }}{{ event_url }}
```

Comment:
Why is it set to this instead of the `event_url` that is available here anyway? This looks like trying to reimplement `reverse`.

Reply:
The reverse only returns the relative path, and hence the url_prefix is added to give the root of the site being hosted. Please correct me if I misunderstood it. 
```
def notify_event_cohost_cohost_status_change(request, cohost, event, status):
    """
    Send email to co-host about their cohost status change for the event.
    """
    subject = f"{settings.SITE_NAME} Event Cohost Status Change"
    context = {
        'name': cohost.get_full_name(),
        'domain': get_current_site(request),
        'url_prefix': get_url_prefix(request),
        'event_title': event.title,
        'event_url': reverse('event_detail', args=[event.slug]),
        'status': status,
        'SITE_NAME': settings.SITE_NAME,
    }
    body = loader.render_to_string('events/email/event_cohost_cohost_status_change.html', context)
    # Not resend the email if there was an integrity error
    send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, [cohost.email], fail_silently=False)
    ```